hack/build.sh

```diff
@@ -45,6 +45,10 @@ for dir in ${dirs}; do
     dirs=( "slave-base ${dirs[@]}")
     break
   fi
+  if [[ "$dir" =~ "agent" ]]; then
+    dirs=( "slave-base ${dirs[@]}")
+    break
+  fi
```

Comment:
if you're building a "slave" image and an "agent" image this is going to end up with a very duplicated array.

just make the existing if do an || check.

Reply:
just pushed